spacy/cli/project/clone.py

```diff
@@ -33,9 +33,25 @@ def project_clone_cli(
     """
     if dest is None:
         dest = Path.cwd() / Path(name).parts[-1]
+    if repo == DEFAULT_REPO:
+        branch = DEFAULT_PROJECTS_BRANCH
```

Comment:
```suggestion
    if repo == DEFAULT_REPO and branch is None:
        branch = DEFAULT_PROJECTS_BRANCH
```
Just in case a user does want to clone from the default repo, but not with the default branch